README.md

```diff
@@ -1,41 +1,52 @@
-nucleus
-==========
+Nucleus
+=======
 
-[![Build Status](https://img.shields.io/travis/mozilla/nucleus/master.svg)](https://travis-ci.org/mozilla/nucleus)
+The publication platform for Mozilla's marketing websites.
 
-[![Coverage status](https://img.shields.io/coveralls/mozilla/nucleus/master.svg)](https://coveralls.io/r/mozilla/nucleus)
-
-Run the tests
--------------
-
-There's a sample test in `nucleus/base/tests.py` for your convenience, that
-you can run using the following command:
+Docker for development
+----------------------
 
-    python manage.py test
+Make sure you have [docker](https://www.docker.com/products/docker-desktop) and 
+[docker-compose](https://github.com/docker/compose). After those are setup and running
+you can use the following commands:
 
-If you want to run the full suite, with flake8 and coverage, you may use
-[tox](https://testrun.org/tox/latest/). This will run the tests the same way
-they are run by [travis](https://travis-ci.org)):
+```bash
+$ # This file must exist and you can customize environment variables for local dev in it
+$ touch .env
+$ # this pulls our latest builds from the docker hub.
+$ # it's optional but will speed up your builds considerably.
+$ docker-compose pull
+$ # get the site up and running
+$ docker-compose up web
+```
 
-    pip install tox
-    tox
+If you've made changes to the `Dockerfile` or the `requirements.txt` files you'll need to rebuild
+the image to run the app and tests:
 
-The `.travis.yml` file will also run [coveralls](https://coveralls.io) by
-default.
+```bash
+$ docker-compose build web
+```
 
-If you want to benefit from Travis and Coveralls, you will need to activate
-both of them in your project.
+Then to run the app you run the `docker-compose up web` command again, or for running tests against your local changes you run:
 
-Oh, and you might want to change the "Build Status" and "Coverage Status" links
-at the top of this file to point to your own travis and coveralls accounts.
+```bash
+$ docker-compose run test
```

Comment:
Should we add the `--rm` flag here and below?

Reply:
Yes indeed! I'll update.